### PR TITLE
Update Windows local mode doc

### PR DESCRIPTION
**Pull request by demianzhang**
Add WindowsLocalMode.md
Update QuickStart and Installation
Add mnist/config_windows.yml,  only change python3 to python

### CODE REVIEW

File: docs/en_US/Installation.md [hunk 1]
```diff
@@ -24,25 +24,25 @@ Currently we support installation on Linux, Mac and Windows.
 
   You can also install NNI in a docker image. Please follow the instructions [here](https://github.com/Microsoft/nni/tree/master/deployment/docker/README.md) to build NNI docker image. The NNI docker image can also be retrieved from Docker Hub through the command `docker pull msranni/nni:latest`.
 
-## **Installation on Windows**
-
+## **Installation on Windows** 
+  When you use powershell to run script for the first time, you need run powershell as Administrator with this command:
```
Comment (PurityFan) [inline docs/en_US/Installation.md]:
```suggestion
  When you use powershell to run script for the first time, you need **run powershell as Administrator** with this command:
```
It's better to bold this part.

File: docs/en_US/WindowsLocalMode.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  powershell ./install.ps1
+  ```
+
+When these things are done, run the **config_windows.yml** file from your command line to start the experiment.
+
+```bash
+    nnictl create --config nni/examples/trials/mnist/config_windows.yml
+```
+For other examples you need to change trial command `python3` into `python` in each example yaml.
+
```
Comment (PurityFan) [inline docs/en_US/WindowsLocalMode.md]: Can you add some errors that user may meet and the suggested solution? (you can collect the information through our dev group chat). And give a link to FAQ.md for further questions.

Comment (demianzhang) [reply docs/en_US/WindowsLocalMode.md]: Thanks for your suggestion, I will add.

File: README.md [hunk 3]
```diff
@@ -100,6 +100,7 @@ Targeting at openness and advancing state-of-art technology, [Microsoft Research
 We encourage researchers and students leverage these projects to accelerate the AI development and research.
 
 ## **Install & Verify**
+Windows local mode installation please check [NNI Windows local mode](docs/en_US/WindowsLocalMode.md)
```
Comment (xuehui1991) [inline README.md]: I think it not clear that you put this sentence here. Could you split the installation for windows and linux?

File: docs/en_US/QuickStart.md [hunk 4]
```diff
@@ -124,16 +127,27 @@ trial:
   codeDir: .
   gpuNum: 0
 ```
+Note:
+* For Windows, you need to change trial command `python3` to `python`
```
Comment (xuehui1991) [inline docs/en_US/QuickStart.md]: Use bold font here？

File: README.md (hunk 3 above)
Comment (demianzhang) [reply README.md]: have changed

**Pull request merged**